sys/conf/ldscript.mips

```diff
@@ -176,6 +180,12 @@ SECTIONS
     }
   .rel.plt       : { *(.rel.plt)		}
   .rela.plt      : { *(.rela.plt)		}
+
+  PROVIDE (_rel_dyn = ALIGN(0x8));
```

Comment:
Uh, no, you shouldn't need these. Look them up in `_DYNAMIC`, it'll give you size information too.

Reply:
I think these were from earlier linker versions. I have now updated it.